index.html

```diff
@@ -0,0 +1,192 @@
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="https://unicons.iconscout.com/release/v4.0.0/css/line.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons+Sharp" />
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Sharp:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+        href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap"
+        rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+</head>
+
+<body>
+    <nav>
+        <div class="top-bar">
+            <ul>
+                <a href="#">
+                    <li><i class="uil uil-facebook-f"></i></li>
+                </a>
+                <a href="#">
+                    <li><i class="uil uil-twitter"></i></li>
+                </a>
+                <a href="#">
+                    <li>English</li>
+                </a>
+                <a href="#">
+                    <li>My Page</li>
+                </a>
+                <a href="#">
+                    <li>Logout</i></li>
+                </a>
+            </ul>
+        </div>
+    </nav>
+    <div class="header">
+        <div class="logo">
+            <a href="#"><img src="./images/logo.png"></a>
+            <a href="#">WHO</a>
+        </div>
+
+        <div class="menu-links">
+            <ul>
+                <a class="menu-link" href="./about.html">
+                    <li>About</li>
+                </a>
+                <a class="menu-link" href="#program">
+                    <li>Program</li>
+                </a>
+                <a class="menu-link" href="#join">
+                    <li>Join</li>
+                </a>
+                <a class="menu-link" href="#sponsor">
+                    <li>Sponsor</li>
+                </a>
+                <a class="menu-link" href="#news">
+                    <li>News</li>
+                </a>
+                <a class="menu-link" href="#speakers">
+                    <li>WHO Campaign</li>
+                </a>
+
+            </ul>
+           <!-- <button id="menu-btn"><span class="material-symbols-sharp" >menu</span></button>
+            <button id="close-btn"><span class="material-symbols-sharp" >close</span></button>-->
+        </div>
+        <button class="humburger">
+            <div class="bar"></div>
+        </button>
+        
+    </div>

```

Comment:
As requested by the previous reviewer, please use the [header](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) element instead of a `div`, to make your project semantically accurate and improve your page accessibility 😉.

You can have something like this:  

```html 
<header> 
  logo 

  <div class="menu-link">  // ideally, this should be a nav element, instead of a div
    links
  </div> 
</header> 
``` 

[OPTIONAL]  Please also update your `class="menu-links"` to a [nav](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) element, to improve your webpage accessibility.